source/templates/stack.slim

```diff
@@ -108,3 +109,4 @@
 							/ a.machine-type-info-link[ng-href="{{ cpuDescriptionURL }}" target="_blank"] == data[:strings][:machine_type][:learn_more_about]
 					.machine-type-warning-wrapper
 						p.notification.warning[ng-if="stackCtrl.workflowMachineType(workflow).id === 'g2-m1.8core'"] == replaced_string(data[:strings][:stack][:m1_elite_xl_beta_warning], [data[:routes][:devcenter][:m1_stacks]])
+						p.notification.warning[ng-if="stackCtrl.workflowMachineType(workflow).id === 'g2-m1-max.5core'"] == replaced_string(data[:strings][:stack][:m1_max_elite_xl_beta_warning], [data[:routes][:devcenter][:m1_stacks]])
```

Comment:
Unrelated to your PR, it would be nice to factor the common bits out from the default and workflow-specific settings. :)